src/main/java/org/openstreetmap/atlas/geography/atlas/raw/slicing/RawAtlasRelationSlicer.java

```diff
@@ -744,11 +745,15 @@ private void patchNonClosedMembers(final Relation relation, final List<RelationM
                         return;
                     }
 
-                    final List<LineString> borderLines;
+                    final Set<LineString> borderLines;
                     try
                     {
-                        borderLines = getCountryBoundaryMap().clipBoundary(relationIdentifier,
-                                polygon);
+                        // The intent is to have the all the unique slices in the same order that
```

Comment:
`is to have the all the unique`: `the` is repeated twice

Reply:
Thanks, will remove.